diagnostics/convective_transition_diag/convecTransCriticalCollapse_util.py

```diff
@@ -511,7 +511,8 @@ def convecTransCriticalCollapse_plot(argsv1,argsv2,argsv3,argsv4,argsv5,argsv6):
         ##### Figure Critical CWV (WC) #####
         fig_obs_wc = mp.figure(figsize=(figsize1/1.5,figsize2/2.6))
 
-        fig_obs_wc.text(s='Critical CWV, Col. Satn., & Critical Col. RH ('+OBS+', '+RES+'$^{\circ}$)', x=0.5, y=1.03, ha='center', va='bottom', transform=fig_obs_wc.transFigure, fontsize=16)
+        #fig_obs_wc.text(s='Critical CWV, Col. Satn., & Critical Col. RH ('+OBS+', '+RES+'$^{\circ}$)', x=0.5, y=1.03, ha='center', va='bottom', transform=fig_obs_wc.transFigure, fontsize=16)
```

Comment:
Is there a need to keep the old line around?

Reply:
Deleted.

Reply:
Deleted.